
# This is a patch for SOAP-Lite-0.710.10 to update it to SOAP-Lite-0.710.10-patched
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "SOAP-Lite-0.710.10\lib\IO\SessionData.pm" "SOAP-Lite-0.710.10-patched\lib\IO\SessionData.pm"
Index: ./lib/IO/SessionData.pm
*** ./lib/IO/SessionData.pm	Tue Sep 29 14:30:08 2009
--- ./lib/IO/SessionData.pm	Tue Feb  9 18:54:08 2010
***************
*** 30,36 ****
          ),
          (eval {require POSIX}
              ? map {
!                 eval { POSIX->can($_) }
                      ? (POSIX->can($_)->() => 1)
                      : ()
                  } @names
--- 30,36 ----
          ),
          (eval {require POSIX}
              ? map {
!                 eval { POSIX->can($_) } && eval { POSIX->can($_)->() }
                      ? (POSIX->can($_)->() => 1)
                      : ()
                  } @names
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Tue Feb  9 18:54:15 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "lib/IO/SessionData.pm" 6749 1265766848 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Tue Feb  9 18:54:15 2010] ####
#### Patch checksum: 55 2018 47838 ####
#### Checksum: 73 2723 40197 ####
